src/models/inspection_time.py

```diff
@@ -0,0 +1,19 @@
+from . import db
+from .abc import BaseModel, MetaBaseModel
+from uuid import uuid4
+
+
+class InspectionTime(db.Model, BaseModel, metaclass=MetaBaseModel):
+    __tablename__ = "inspection_times"
+
+    id = db.Column(db.String(36), primary_key=True)
```

Comment:
jak w poprzednim komentarzu uuid byłoby lepsze od stringa, pewnie się da to jakoś zdefiniować